WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift

```diff
@@ -441,6 +454,9 @@ extension ReaderFollowedSitesViewController: WPTableViewHandlerDelegate {
     }
 
     func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+
+        guard showsSectionTitle else { return nil }
```

Comment:
The `return` should be on a new line. ([style guide](https://github.com/wordpress-mobile/swift-style-guide))